namui/src/namui/render/rendering_tree/mod.rs

```diff
@@ -41,83 +41,77 @@ impl SpecialRenderingNode {
 /// - events = Reverse post-order (RLN)
 /// reference: https://en.wikipedia.org/wiki/Tree_traversal
 impl RenderingTree {
-    pub(crate) fn draw(&self, namui_context: &NamuiContext) {
+    pub(crate) fn draw(&self) {
```

Comment:
I found that they don't need context when they draw.
Previously it was needed to use surface, but from now on, they use system so they don't need it.